src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const populationCountry = document.querySelectorAll('span');
+const len = document.querySelectorAll('.population').length;
+
+const avg = document.querySelector('.average-population');
+
+avg.textContent = len;
+
+const total = document.querySelector('.total-population');
+
+let massPupulation = [];
+
+for (let i = 0; i < len; i++) {
+  let str = populationCountry[i].textContent;
+
+  str = str.replaceAll(',','');
+
+  massPupulation.push(Number(str));
+}
+
+let totalSum = 0;
+
+for (const elem in massPupulation) {
```

Comment:
The `for...in` loop is not the best choice for iterating over arrays. Consider using a `for...of` loop or a standard `for` loop, as `for...in` is intended for iterating over object properties and may lead to unexpected behavior with arrays.